src/streamlink/plugins/dogus.py

```diff
@@ -12,14 +14,18 @@
 from streamlink.plugin import Plugin, pluginmatcher
 from streamlink.plugin.api import validate
 from streamlink.stream.hls import HLSStream
-
+from streamlink.utils.url import update_scheme
```

Comment:
Missing new line after the imports. Imports and class definitions are separated by two lines.